server/src/main/java/org/apache/iotdb/db/engine/flush/MemTableFlushTask.java

```diff
@@ -235,6 +235,8 @@ public void run() {
             ChunkWriterImpl chunkWriter = (ChunkWriterImpl) ioMessage;
             chunkWriter.writeToFileWriter(this.writer);
           } else {
+            this.writer.setMinPlanIndex(memTable.getMinPlanIndex());
```

Comment:
why add `min/max planIndex` on ChunkGroupFooter?

Reply:
It mainly used for a follower to catch up to the leader when doing snapshot task to filter tsfiles,  for details please see https://issues.apache.org/jira/browse/IOTDB-918